src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1648,16 +1649,18 @@ class spell_dru_wild_growth : public SpellScript
 
     bool Validate(SpellInfo const* spellInfo) override
     {
-        if (!ValidateSpellEffect({ { spellInfo->Id, EFFECT_2 } }) || spellInfo->GetEffect(EFFECT_2).IsEffect() || spellInfo->GetEffect(EFFECT_2).CalcValue() <= 0)
-            return false;
-        return true;
+        return ValidateSpellEffect({ { spellInfo->Id, EFFECT_1 }, { SPELL_DRUID_TREE_OF_LIFE, EFFECT_2 } })
+            && ValidateSpellInfo({ SPELL_DRUID_TREE_OF_LIFE });
     }
 
     void FilterTargets(std::list<WorldObject*>& targets)
     {
         targets.remove_if(RaidCheck(GetCaster()));
```

Comment:
This is wrong up to this day. Wild Growth (like many other AoE healing spells) do not remove targets that are outside the caster's group or raid group, they only prefer targets within.

Reply:
that spell is outdated (effect 2 doesnt exist) but that should be another pr

Reply:
This PR is focused on Incarnation tree of life, not anything else, its fine to leave this as it is now